test/models/artefact_test.rb

```diff
@@ -231,6 +231,19 @@ class ArtefactTest < ActiveSupport::TestCase
     end
   end
 
+  should "validate redirect_url" do
+    artefact = FactoryGirl.create(:artefact)
+
+    artefact.redirect_url = "foobar"
+    refute artefact.valid?
+
+    artefact.redirect_url = "/foobar"
+    assert artefact.valid?
+
+    artefact.redirect_url = "http://foo.bar/"
+    assert artefact.valid?
```

Comment:
It'd support cases like these: https://github.gds/gds/router-data/blob/master/data/external-redirects.csv